### ISSUE TITLE
Makefile missing

### ISSUE THREAD

**Issue by bulvara** [gfx/Perl-Data-Util#19]
Dear Fuji,

I'm trying to update the [ArchLinux PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=perl-data-util).

Sadly I can't figure out how to install it after successfully building with
```
    /usr/bin/perl Build.PL
    /usr/bin/perl Build
```

There is no Makefile for `make install` anymore.

BTW: Are perl-hash-util-fieldhash-compat and perl-scope-guard still needed?

**Comment by syohex** [comment]
You can install by following commands.

```
perl Build.PL
./Build
./Build install
```

`perl-hash-util-fieldhash-compat` is necessary for the pure Perl implementation. `perl-scope-guard` is only necessary for testing, so I fixed it at #20.

**Comment by bulvara** [comment]
Thanks
